rules-tests/Php74/Rector/Property/TypedPropertyRector/Source/SomeParent.php

```diff
@@ -15,4 +15,9 @@ abstract class SomeParent
      * @var string
      */
     protected $typedName;
+
+    /**
+     * @var bool
+     */
+    protected $active = false;
```

Comment:
this should be typed `bool` already.

Reply:
no, in that case I'm simulating if the child gets read first from the file system

Reply:
that seems can't happen, if the child is changed, while parent not changed yet, that seems cause `Fatal error`, see https://3v4l.org/C2u0u

The change of: 

```php
protected bool $anAlreadyReplacedProperty = true;
```

seems fine, while `$active` should be kept as is if parent didn't change yet.

Reply:
That is what I was trying to explain. the already replaced property must be with type bool, otherwise it is not in sync with the parent which will crash. In that case Rektor is doing something weird.
The replacement of active property would be awesome if that could work but it's complicated, I know

Reply:
in that case, you can try `composer dump-autoload -o`, if still happen, you can skip that like described in documentation 

https://github.com/rectorphp/rector/blob/70acf467c720c96220ad67f60c05467deb3dca20/docs/static_reflection_and_autoload.md#troubleshooting

Reply:
As you can see from the test results rector was able to do something and this was exactly the case that I was running into in my own project that the child class got "reverted back" which must not happen! This must be fixed.

Reply:
As I stated before, the only valid test case is the `protected bool $anAlreadyReplacedProperty = true;` property so child should follow parent typed, the `protected $active = false;` is invalid as parent still has no type, which already handle as expected.

```
+    /**
+     * @var bool
+     */
+    protected $active = true;
```